packaging/wrappers/grafana-cli.wrapper

```diff
@@ -0,0 +1,58 @@
+#! /usr/bin/env bash
+
+# Wrapper for the grafana-cli binary
+# grafana-cli.wrapper calls the grafana-cli binary with default configuration options. It ensures we set the system-wide Grafana configuration that was bundled with the Debian and RPM packages.
+
+DEFAULT=/etc/default/grafana
+
+GRAFANA_HOME=/usr/share/grafana
+CONF_DIR=/etc/grafana
+DATA_DIR=/var/lib/grafana
+PLUGINS_DIR=/var/lib/grafana/plugins
+LOG_DIR=/var/log/grafana
+
+CONF_FILE=$CONF_DIR/grafana.ini
+PROVISIONING_CFG_DIR=$CONF_DIR/provisioning
+
+EXECUTABLE=/usr/sbin/grafana-cli
+
+if [ ! -x $EXECUTABLE ]; then
+ echo "Program not installed or not executable"
+ exit 5
+fi
+
+if [ $# -eq 0 ]; then
+  eval $EXECUTABLE
+  exit 0
+fi
+
+# overwrite settings from default file
+if [ -f "$DEFAULT" ]; then
+  . "$DEFAULT"
+fi
+
+OPTS="--homepath=${GRAFANA_HOME} --config=${CONF_FILE}"
+CFG_OPTS="cfg:default.paths.provisioning=$PROVISIONING_CFG_DIR \
+      cfg:default.paths.data=${DATA_DIR} \
+      cfg:default.paths.logs=${LOG_DIR} \
+      cfg:default.paths.plugins=${PLUGINS_DIR}"
+
+COMMAND=$1
+SUBCOMMAND=$2
+
+# We special case admin commands as is they are the only ones the that supports --homepath and --config as flags.
```

Comment:
could we pass those args regardless if the command needs them? Just to make the script simpler.

Reply:
Sadly 😢 Since the configuration overrides are not flags the CLI ends up thinking they are commands.